gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/statistics/ConnectedComponentsDistributionAsValues.java

```diff
@@ -24,19 +24,19 @@
 /**
  * Computes the weakly connected components of a graph structure. Uses the gradoop wrapper
  * {@link ValueWeaklyConnectedComponents} of Flinks ConnectedComponents.
- *
```

Comment:
Please restore all changes of this file. There are no changes needed in this class.